definitions/json/CaseTypeTab/CaseTypeTab-Multiples-nonprod.json

```diff
@@ -1,4 +1,21 @@
 [
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "Channel": "CaseWorker",
+    "TabID": "Notes",
+    "TabLabel": "Notes",
+    "CaseFieldID": "multiplesNotes",
+    "TabDisplayOrder": 2
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
```

Comment:
Is this for note history ?

Reply:
I'm not sure what it's for. Ticket says 'Add the standard case Notes tab'